Gruntfile.coffee

```diff
@@ -140,6 +140,77 @@ backendThirdPartyBuildPaths = [
   'uproxy-lib/cloud/social'
 ]
 
+getWithBasePath = (files, base = '') ->
```

Comment:
A comment would be good. Also, consider moving this closer to the other functions, e.g. `compileTypescript`.